src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+
+        UserService userService = new UserService();
+        if (userService.findByEmail(email) == null) {
+            return false;
+        }
+        return password.equals(userService.findByEmail(email).getPassword());
```

Comment:
This expression is overcomplicated. I suggest you to make this operation in 2 lines to make code more readable